lucene/ivy-versions.properties

```diff
@@ -116,7 +117,7 @@ org.apache.calcite.version = 1.13.0
 /org.apache.commons/commons-math3 = 3.6.1
 /org.apache.commons/commons-text = 1.4
 
-org.apache.curator.version = 2.8.0
+org.apache.curator.version = 2.13.0
```

Comment:
This version shades guava - don't end up with Guava issues down the line and matches the Hadoop 3.2.0 curator version.